desktop/app/console-helper.js

```diff
@@ -1,27 +1,28 @@
 import {ipcMain, ipcRenderer} from 'electron'
 import util from 'util'
 
+const methods = ['log', 'error', 'info']
```

Comment:
Ensure we only ever call the original console commands and not old versions of our redirects (over and over)